src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -72,9 +72,14 @@ object Import {
       """Declares endpoints. The default is Map("web" -> Endpoint("http", 0, Set("http://:9000"))) where the service name is the name of this project. The "web" key is used to form a set of environment variables for your components. For example you will have a `WEB_BIND_PORT` in this example."""
     )
 
+    val checkInitialDelay = SettingKey[Int](
```

Comment:
I think that we should always use a type where we can. Adding an additional import here is fine. This is not a common setting either.

The URI params are more to do with the check command itself, and not with sbt settings, so I'm comfortable with the difference.